frontend/src/app/testQueue/TestCardForm/diseaseSpecificComponents/aoeQuestionComponents/GenderOfSexualPartnersAoe.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import { SensitiveTopicsTooltipModal } from "../SensitiveTopicsTooltipModal";
+import MultiSelect from "../../../../commonComponents/MultiSelect/MultiSelect";
+import { GENDER_IDENTITY_VALUES } from "../../../../constants";
+import { AoeQuestionProps } from "../aoeUtils";
+
+export const GenderOfSexualPartnersAoe = ({
+  testOrderId,
+  responses,
+  hasAttemptedSubmit,
+  onResponseChange,
+  required = false,
+}: AoeQuestionProps) => {
+  const onSexualPartnerGenderChange = (selectedItems: string[]) => {
+    onResponseChange({
+      ...responses,
+      genderOfSexualPartners: selectedItems,
+    });
+  };
+  const selectedGenders: string[] = [];
+  responses.genderOfSexualPartners?.forEach((g) => {
+    if (g) {
+      selectedGenders.push(g);
+    }
+  });
+  const isGenderOfSexualPartnersAnswered =
+    !!responses.genderOfSexualPartners &&
+    responses.genderOfSexualPartners.length > 0;
+  const showGenderOfSexualPartnersError =
+    hasAttemptedSubmit && !isGenderOfSexualPartnersAnswered;
+
+  return (
+    <MultiSelect
+      name={`sexual-partner-gender-${testOrderId}`}
+      options={GENDER_IDENTITY_VALUES}
+      onChange={onSexualPartnerGenderChange}
+      initialSelectedValues={selectedGenders}
+      label={
+        <>
+          What is the gender of their sexual partners?{" "}
+          <span className={"text-base-dark"}>(Select all that apply.)</span>
+        </>
+      }
+      required={required}
+      hintText={<SensitiveTopicsTooltipModal showSyphilis={false} />}
```

Comment:
(non-blocking) when we go back to swap the usages of the gender of sexual partners question with this question we could also make the `showSyphilis` props more generic

Reply:
Yep! That's one of the changes I'll have in the follow up PR for the sensitive topics modal, one of which will also be moving it out of the question component here and into the disease's specific AOE form component so it can be composed at that level and then passed into whichever AOE question needs it as hint text